docs/02-developers/05-smart-contracts/05-foundry/advanced-development-patterns.md

```diff
@@ -0,0 +1,951 @@
+# Advanced Smart Contract Development and Deployment with Foundry on Rootstock
+
+---
```

Comment:
@gboigwe Metadata is not formatted properly: https://devportal-rootstock-git-fork-gboigwe-feat-f-74737b-iov-labs-mkt.vercel.app/developers/smart-contracts/foundry/advanced-development-patterns/

Kindly refer to existing pages for the right metadata values, and remove location in the metadata

<img width="1726" height="988" alt="Screenshot 2025-07-22 at 12 34 32" src="https://github.com/user-attachments/assets/4e36ac07-dd5a-4785-b233-4e47848d2046" />